### PR TITLE
George - CHARMS RAS Screener Verification Updates

### PR DESCRIPTION
**Pull request by georgeb79**

### CODE REVIEW

File: src/test/java/ServiceNow/CHARMS/Features/RasopathyStudy/RASSurvey.feature
[the remark this answers is not in the document]
Comment (georgeb79) [reply src/test/java/ServiceNow/CHARMS/Features/RasopathyStudy/RASSurvey.feature:1]: This is an older scenario that is not used anymore. I just had to remove the @Regression so it won't run when running regression.

**Review by puzikovam2 — APPROVED** [review]
approved by puzikovam2

**Review by andreyeve — APPROVED** [review]

**Review by sarwarahmed1 — APPROVED** [review]

File: src/test/java/ServiceNow/CHARMS/Features/RasopathyStudy/RASSurvey.feature
[the remark this answers is not in the document]
Comment (matakevin) [reply src/test/java/ServiceNow/CHARMS/Features/RasopathyStudy/RASSurvey.feature:1]: We should still use a deprecated tag or something similar when marked a test case outdated.  We can discuss what we can tag in the next meeting.  For now, this is fine

File: src/test/java/ServiceNow/CHARMS/Features/RasopathyStudy/RAS Screener Data Verification.feature
Comment (matakevin) [inline src/test/java/ServiceNow/CHARMS/Features/RasopathyStudy/RAS Screener Data Verification.feature:1]: Is there a ticket number for this test case?

**Review by matakevin — APPROVED** [review]
Left some comments but none are too pressing so aside from those, LGTM

File: src/test/java/ServiceNow/CHARMS/Features/RasopathyStudy/RASSurvey.feature
[the remark this answers is not in the document]
Comment (georgeb79) [reply src/test/java/ServiceNow/CHARMS/Features/RasopathyStudy/RASSurvey.feature:1]: I agree. I don't know,  Diego or Sonika may reuse it in the future. I just removed the @Regression tag so I could run my regression

File: src/test/java/ServiceNow/CHARMS/Features/RasopathyStudy/RAS Screener Data Verification.feature
Comment (georgeb79) [reply src/test/java/ServiceNow/CHARMS/Features/RasopathyStudy/RAS Screener Data Verification.feature:1]: Is not part of a regression suite. Is used just to run verification on 1 scenario part of a bigger flow. Is still a work in progress.

**Pull request merged**